docs/getting_started/deployment.md

```diff
@@ -267,6 +267,286 @@ spu = sf.SPU(cluster_def=cluster_def)
 
 Due to the complexity of the Ray communication protocol, including cross-node mutual access and using many ports, it's really complex and error-prone to deploy multiple containers on multiple machines using the bridge network to build a Ray cluster for simulation. Ray's official support for this is not good, so we do not recommend this deployment mode.
 
+### Deploy SecretFlow in WSL(Windows Subsystem for Linux) with simulation mode
+
+If you use SecretFlow in WSL2, you could deploy SecretFlow with different distributions in WSL2.
```

Comment:
Why is there emphasis on two different Linux distribution packages here? WSL does not allow run the same distribution more than once at the same time?

Reply:
In Microsoft Store, if you have install one Linux distribution package such as Ubuntu 22.04.2 LTS, you could not  install another independent Ubuntu 22.04.2 LTS, which means that you run the same distribution more than once at the same time is equal to that you open more than one terminal based on the same distribution.In this case, I emphasis on two different Linux distribution packages here to simulate two machine host which have indenpendent file system and indenpendent users but same IP address and then we deploy SecretFlow in WSL with simulation mode but not single machine mode.

Reply:
I C.

Reply:
Strictly speaking, Maybe we can use migration,import and rename of wsl to realize the deploying SecretFlow in WSL with same distribution,  but it maybe not suitable for guide document for the reason that it bring extra complex operation and other issue.
In this case, I recommend two different Linux distribution.
migration,import and rename of wsl ref to: 
https://blog.csdn.net/weixin_43408232/article/details/128606034?spm=1001.2014.3001.5501
here mention how to migrate wsl.